packages/uma-auth-client/src/hooks/useOAuth.tsx

```diff
@@ -104,6 +105,16 @@ export const useOAuth = create<OAuthState>()(
         }
         return true;
       },
+      clearUserAuth: () => {
+        set({
+          token: undefined,
+          nwcConnectionUri: undefined,
+          nwcExpiresAt: undefined,
+          codeVerifier: undefined,
+          csrfState: undefined,
+          uma: undefined,
+        });
```

Comment:
Could this just be `set({})` or did ts not like that for some reason?

Reply:
If the property isn't specified zustand doesn't update the state for it, so in this case we need to manually specify undefined

we also need to keep the authConfig, so this isn't actually clearing everything